git_machete/client.py

```diff
@@ -91,7 +91,7 @@ def get_childless_branches(self) -> List[LocalBranchShortName]:
     def expect_in_managed_branches(self, branch: LocalBranchShortName) -> None:
         if branch not in self.managed_branches:
```

Comment:
Whoops not sure where exactly is the source of the problem... but 

<img width="264" alt="image" src="https://user-images.githubusercontent.com/3383210/213559395-bbf4d29a-45b7-4924-845a-4ebd077793e6.png">

some of `github` subcommands started adding ANSI escapes to annotations, which isn't expected... pls fix